vdev/src/commands/mod.rs

```diff
@@ -112,3 +112,29 @@ macro_rules! script_wrapper {
         }
     };
 }
+
+/// This macro creates a wrapper for an existing package script. If the command is run from within
+/// the CI environment (by checking the CI env var), the repo directory is marked as safe, to
+/// satisfy git permissions requirements.
+#[macro_export]
+macro_rules! package_script_wrapper {
+    ( $mod:ident = $doc:literal => $script:literal ) => {
+        paste::paste! {
+            mod $mod {
+                #[doc = $doc]
+                #[derive(clap::Args, Debug)]
+                #[command()]
+                pub(super) struct Cli {
+                    args: Vec<String>,
+                }
+
+                impl Cli {
+                    pub(super) fn exec(self) -> anyhow::Result<()> {
+                        $crate::util::mark_safe_git_repo();
```

Comment:
This macro appears to be identical to the above except for this line. Any chance of merging the two, maybe by using a null expression where this would go?

Reply:
attempted to resolve in [2b0da0b](https://github.com/vectordotdev/vector/pull/16281/commits/2b0da0b0b2ce4cfdbd67b51e77063b67b0023959)